%YAML:1.2
body2world_pose: !!opencv-matrix
  rows: 4
  cols: 4
  dt: d
  data: [0.6168037504345139, -0.3803477656101839, 0.6891216950911201, 0.12038990033344923, -0.6549149383752451, -0.7336405646531414, 0.18126760711271253, -0.02804149511904546, 0.43662290015855443, -0.5631226323730564, -0.701607685224708, 1.087785086204046, 0.0, 0.0, 0.0, 1.0]

















